setup.cfg

```diff
@@ -13,10 +13,10 @@ classifiers = License :: OSI Approved :: BSD License
               Programming Language :: Python :: 2
               Programming Language :: Python :: 2.7
               Programming Language :: Python :: 3
-              Programming Language :: Python :: 3.4
```

Comment:
Why is this?  Travis-CI on your branch - still has tests for CPython 3.4.

Reply:
Python 3.4 and 3.5 are end-of-life.
https://devguide.python.org/devcycle/#end-of-life-branches
https://devguide.python.org/#status-of-python-branches

Reply:
2.7 - too, and?